src/covid19sim/hydra-configs/constants.yaml

```diff
@@ -0,0 +1,109 @@
+# Time.
```

Comment:
A discussion around this seems to be inclined towards making this a `python` file. Is that possible?